src/compas_fab/robots/robot.py

```diff
@@ -110,6 +112,49 @@ def __from_data__(cls, data):
         robot.attributes = attributes
         return robot
 
+    @classmethod
+    def from_urdf(cls, urdf_filename, srdf_filename=None, local_package_mesh_folder=None, client=None):
+        # type: (str, Optional[str], Optional[str], Optional[ClientInterface]) -> Robot
```

Comment:
`ClientInterface` is not imported, is it? Does this work? Can you do a fully-qualified type here (ie. `compas_fab.backends.interfaces.ClientInterface`) to avoid importing only for the purpose of type checking?